tests/unitTests/borgcron.sh

```diff
@@ -286,7 +286,7 @@ testNotRetry(){
 	addFakeBorgCommand 'exit 2'
 
 	# TODO: change to "assertFalse" when exit code is implemented
```

Comment:
Yes, that's what you should do, but as you did it now, you can also remove the TODO message.

Reply:
✅